apps/system/fxa/test/marionette/fxa_launch_test.js

```diff
@@ -19,7 +19,7 @@ marionette('Firefox Accounts Launch Tests', function() {
             config.SERVER_PATH
           },
           settings: {
-            'lockscreen.enabled': false
+            'ftu.manifestURL': FxA.FTU_ORIGIN + '/manifest.webapp'
```

Comment:
Mainly because the test already references this origin as the FTU origin, and was more consistent in this file.